src/org/openlcb/cdi/swing/CdiPanel.java

```diff
@@ -221,7 +242,8 @@ private void createSensorCreateHelper() {
         JPanel lineHelper = new JPanel();
         lineHelper.setAlignmentX(Component.LEFT_ALIGNMENT);
         lineHelper.setLayout(new BoxLayout(lineHelper, BoxLayout.X_AXIS));
-        lineHelper.setBorder(BorderFactory.createTitledBorder("User name"));
+        // titled border comes from some else. Not here.
+        //lineHelper.setBorder(BorderFactory.createTitledBorder("User name"));
```

Comment:
Restored. However, I see no change in the display. Titled border is "Description" with "User name of this output" underneath.  
Likewise, there is is a titled border "Event On" rather than what the code has as "Event Id for Active / Thrown".